### ISSUE TITLE
dependency resolution error with next v13 and contentlayer

### ISSUE THREAD

**Issue by timlrx** [timlrx/pliny#78]
**To Reproduce**
Bootstrapping a new package with npm or running npm install in the repo will throw the following error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: next-contentlayer@0.2.8
npm ERR! Found: next@13.0.2
npm ERR! node_modules/next
npm ERR!   next@"13.0.2" from the root project
npm ERR!   peer next@"*" from next-themes@0.2.1
npm ERR!   node_modules/next-themes
npm ERR!     next-themes@"^0.2.0" from the root project
npm ERR!     next-themes@"^0.2.0" from pliny@0.0.9
npm ERR!     node_modules/pliny
npm ERR!       pliny@"0.0.9" from the root project
npm ERR!   1 more (pliny)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer next@"^12" from next-contentlayer@0.2.8
npm ERR! node_modules/next-contentlayer
npm ERR!   next-contentlayer@"0.2.8" from the root project
npm ERR!   next-contentlayer@"0.2.8" from pliny@0.0.9
npm ERR!   node_modules/pliny
npm ERR!     pliny@"0.0.9" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: next@12.3.2
npm ERR! node_modules/next
npm ERR!   peer next@"^12" from next-contentlayer@0.2.8
npm ERR!   node_modules/next-contentlayer
npm ERR!     next-contentlayer@"0.2.8" from the root project
npm ERR!     next-contentlayer@"0.2.8" from pliny@0.0.9
npm ERR!     node_modules/pliny
npm ERR!       pliny@"0.0.9" from the root project
```

**Workaround**
Manually install with `npm install --legacy-peer-deps` or use yarn in the meantime until `next-contentlayer` updates its dependency

**Comment by timlrx** [comment]
Seems like there is no Yarn equivalent for npx. The Yarn team thought npx was enough. Source:
https://stackoverflow.com/questions/66496166/is-there-any-yarn-equivalent-of-npx-preact-create

You could very well use it to bootstrap a starter which uses yarn though with `pliny new --template=starter-blog --yarn my-blog`

**Comment by bravo-kernel** [comment]
Looks fixed for `npm` in 0.0.10 💃
